hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/DeleteBlocksCommandHandler.java

```diff
@@ -121,7 +126,14 @@ public DeleteBlocksCommandHandler(OzoneContainer container,
         dnConf.getBlockDeleteThreads(), threadFactory);
     this.deleteCommandQueues =
         new LinkedBlockingQueue<>(dnConf.getBlockDeleteQueueLimit());
-    handlerThread = new Daemon(new DeleteCmdWorker());
+    long interval = this.conf.getLong(HddsConfigKeys.
+        HDDS_COMMAND_DELETE_WORKER_INTERVAL,
+        HddsConfigKeys.HDDS_COMMAND_DELETE_WORKER_INTERVAL_DEFAULT);
+    Preconditions.checkState(interval > 0,
+        "The value of %s must be greater than 0.",
+        HddsConfigKeys.HDDS_COMMAND_DELETE_WORKER_INTERVAL);
```

Comment:
After moving the property to `DatanodeConfiguration`, this check can be moved to `validate()` there:

https://github.com/apache/ozone/blob/366c2fb57ae15acbb4c357086328ddec2faeb4f3/hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java#L535-L543